### DISCUSSION TITLE
局域网下无法预览office类文件

### DISCUSSION THREAD

**Discussion by townsinger** [AlistGo/alist#816] [Q&A]
各位大佬，win下搭建Alist,挂在本地硬盘。通过内网IP:5244访问，图片都可正常预览，但是word excel都显示错误，无法预览。是因为局域网无法联通office api导致的吗 ？有其他解决的方案吗？感谢各位大佬指点！

**Comment by StudentWeis** [comment]
看这里 https://github.com/alist-org/alist/discussions/594
